packages/legend-graph/src/graphManager/action/execution/ExecutionResult.ts

```diff
@@ -23,6 +23,10 @@ export enum BuilderType {
   JSON_BUILDER = 'json',
 }
 
+export enum EXECUTION_SERIALIZATION_FORMAT {
```

Comment:
Let's add a `TODO` here linking to the issue where we refactor this to use `external format`